pandas/tests/scalar/timestamp/test_unary_ops.py

```diff
@@ -297,6 +298,7 @@ def test_round_implementation_bounds(self):
         with pytest.raises(OverflowError, match=msg):
             Timestamp.max.ceil("s")
 
+    @pytest.mark.xfail(not IS64, reason="Failing on 32 bit build", strict=False)
```

Comment:
instead what if we just adjust the bounds by 1e9?

Reply:
So just to confirm

```
@given(val=st.integers(iNaT + 1 + 1e9, lib.i8max - 1e9))
```

Reply:
i guess technically should use `10**9` but yah

Reply:
I had to cap the max and min values to `np.iinfo(np.in32)` for the 32 bit build instead

Reply:
that seems like something really is wrong on the 32bit build, so maybe an xfail is appropriate after all?

Reply:
Okay. I will change it back to `xfail(strict=False)` because I think the hypothesis still doesn't fail this consistently with certain params.